app/models/cart.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Cart < ApplicationRecord
+  has_many :line_items, dependent: :destroy
+  has_many :products, through: :line_items
+
+  def total
+    line_items.inject(0) { |sum, item| item.total_price + sum }
+  end
+
+  def add_product(product)
```

Comment:
nie podoba mi sie ta metoda. Add sugeruje, ze cos zostanie dodane, a tutaj jedynie to buildujesz. Moze wynies to do interaktora / SO ?

Reply:
przerobione przy pomocy interactora